_pages/dev/lazy-loading-guide.md

```diff
@@ -24,27 +24,60 @@ Code splitting is a technique that has to be done at application build time. Cod
 
 The following sections offer some important information about how lazing loading works in Spartacus.
 
-### Define Dynamic Imports Only in the Main Application
+### Defining Dynamic Imports Only in the Main Application
 
 Dynamic imports, a technique that is used to facilitate lazy loading and that also allows code splitting, can only be used in the main application. It is not possible to define dynamic imports in the prebuilt libraries.
 
-This is an unfortunate limitation, which results in some application code that must be added by customers. Although the amount of custom code is limited to the bare minimum, we'll add a feature in a future version of the schematics library to automatically add lazy loaded modules.
+This is an unfortunate limitation, which results in some application code that must be added by customers. Although the amount of custom code is limited to the bare minimum, we'll add a feature in a future version of the schematics library to automatically add lazy-loaded modules.
 
-### Avoiding Static Imports for Lazy Loaded Code
+### Avoiding Static Imports for Lazy-Loaded Code
 
 To make code spitting possible, your static JavaScript code (the main app bundle) should not have any static imports to code that you want to lazy load. The builder will notice that the code is already included, and as a result, will not generate a separate chunk for it. This is especially important in the case of importing symbols from libraries.
 
-At the time of writing (Angular 9 and Angular 10), mixing static imports with dynamic imports for the same library entry point, even for distinct symbols, will break lazy loading and tree shaking for this library entry point. This includes the whole entry point statically into the build. For this reason, it is highly recommended you create separate entry points for the code that has to be loaded statically, and for the code that can be loaded lazily.
+At the time of writing (Angular 9 and Angular 10), mixing static imports with dynamic imports for the same library entry point, even for distinct symbols, will break lazy loading and tree shaking for this library entry point. If you were to do this, it would statically include the entire entry point in the build. For this reason, it is highly recommended that you create one entry point for the code that has to be loaded statically, and a separate entry point for the code that can be loaded lazily.
 
```

Comment:
Can you verify the last sentence in this paragraph? I could have also written:

"For this reason, it is highly recommended that you create one set of entry points for the code that has to be loaded statically, and a separate set of entry points for the code that can be loaded lazily."

I was trying to slightly improve the structure of the original sentence, but then it requires these extra details (one entry point? many entry points?). I  wasn't sure how many entry points there could be, both for statically-loaded code and for lazy-loaded code, and I didn't want to introduce inaccuracies in the text, so I wanted to draw your attention to this modification that I made... 🙂

Reply:
We should not suggest that customer can create a specific number of entry points, this suggests that user/customer should create one for eagerly loaded parts and one for lazy, in practice, the important take should be that he shouldn't mix it in one endpoint, thus keep it in separate endpoints (no matter how many he will have them).

Reply:
Perfect! Thank you very much 🙂